.github/workflows/node.js.yml

```diff
@@ -26,6 +26,5 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-    - run: npm ci
```

Comment:
Why aren't we using CI here? It should be better than regular install for automation.